src/app/systelab-components/twolist/two-list.component.html

```diff
@@ -7,22 +7,26 @@
         <div class="slab-twolistbox mt-2 slab-flex-1">
             <div class="slab-twolistboxdiv">
                 <ul class="slab-twolistboxrow list-group">
-                    <li *ngFor="let item of (available | datafilter:firstListSearch)"
-                        [ngClass]="{'slab-twolistboxrow slab-twolistboxrowselected':item.selected,'slab-twolistboxrow':!item.selected}"
-                        (click)="selectAvailableItem(item, $event)">
+                    <li *ngFor="let item of (available | datafilter:firstListSearch)" [ngClass]="{'slab-twolistboxrow slab-twolistboxrowselected':item.selected,'slab-twolistboxrow':!item.selected}"
+                        (click)="selectAvailableItem(item, $event)" (dblclick)="moveSelectedAvailableItem(item, $event)">
                         {{item.displayName}}
                     </li>
                 </ul>
             </div>
         </div>
     </div>
-    <div class="slab-twolistbuttondiv">
-        <button type="button" class="slab-twolistbutton btn icon-chevron-right" (click)="add()" title="Add selected"
-                [disabled]="selected.available.length == 0">
+    <div class="slab-twolistbuttondiv  d-flex flex-column flex-nowrap">
```

Comment:
If flex classes are added here then it can be removed from _two-list.scss in class slab-twolistbuttondiv

Reply:
I have removed the class "slab-twolistbutton" of the buttons and also the flex styles in the scss. The others styles we need to prevent small lists and set always the buttons after the filter boxes (vertical alignment)